wormbot:
  loop_rate: 30